plugin/pkg/shoot/validator/admission.go

```diff
@@ -714,6 +715,36 @@ func (c *validationContext) validateAdmissionPlugins(a admission.Attributes, sec
 	return allErrs
 }
 
+func (c *validationContext) validateOIDCConfig(a admission.Attributes) field.ErrorList {
```

Comment:
You could add a comment here why this is part of the admission plugin validation in `plugin/pkg` and not part of the static API validation in `pkg/apis`.